curator/templates/single.html

```diff
@@ -1,28 +1,121 @@
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:100%"><!-- wp:post-excerpt {"textAlign":"center","fontSize":"x-large"} /--></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:post-content {"layout":{"inherit":true}} /--></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px","bottom":"30px"}},"border":{"top":{"color":"var:preset|color|foreground","width":"2px"},"bottom":{"color":"var:preset|color|foreground","width":"2px"}}},"layout":{"inherit":false}} -->
+<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--foreground);border-top-width:2px;border-bottom-color:var(--wp--preset--color--foreground);border-bottom-width:2px;padding-top:30px;padding-bottom:30px"><!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group"><!-- wp:post-navigation-link {"type":"previous","label":"\u003cstrong\u003ePREVIOUS POST\u003c/strong\u003e","fontSize":"large"} /-->
+
+<!-- wp:post-navigation-link {"label":"\u003cstrong\u003eNEXT POST\u003c/strong\u003e","fontSize":"large"} /--></div>
+<!-- /wp:group --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
 
 <!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group">
-<!-- wp:post-title /-->
-</div>
+<div class="wp-block-group"><!-- wp:post-comments-form /--></div>
 <!-- /wp:group -->
 
-<!-- wp:group {"tagName":"main"} -->
-<main class="wp-block-group">
-<!-- wp:post-featured-image {"align":"full"} /-->
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"fontSize":"x-large"} -->
+<p class="has-x-large-font-size">Comments</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"fontSize":"x-large"} -->
+<p class="has-x-large-font-size">(</p>
+<!-- /wp:paragraph -->
 
-<!-- wp:post-content {"layout":{"inherit":true}} /-->
+<!-- wp:post-comments-count {"textAlign":"center","fontSize":"x-large"} /-->
```

Comment:
There's quite a bit of space between the parenthesis and comments count here — any way we can reduce that? 

<img width="326" alt="Screen Shot 2022-05-10 at 11 27 47 AM" src="https://user-images.githubusercontent.com/5375500/167665886-48f90aa3-6ee3-4509-bd5f-6896d9014ca8.png">